[CI] UpdateSettingsRequestStreamableTests mutate failure

The error appears to be that for certain seeds the `mutate()` function has no effect
```
UpdateSettingsRequestStreamableTests.testEqualsAndHashcode <<< FAILURES!
   > Throwable #1: java.lang.AssertionError: UpdateSettingsRequest mutation should not be equal to original
   > Expected: not <indices : [index-kcp],{"HJS":{"Gqj":{"KILj":{"wkCM":{"Fhk":"pw"}}}}}>
   >      but: was <indices : [index-kcp],{"HJS":{"Gqj":{"KILj":{"wkCM":{"Fhk":"pw"}}}}}>
```

Reproduces:
```
 ./gradlew :server:test \
  -Dtests.seed=E1A4DF226F82345F \
  -Dtests.class=org.elasticsearch.action.admin.indices.settings.put.UpdateSettingsRequestStreamableTests \
  -Dtests.method="testEqualsAndHashcode" \
  -Dtests.security.manager=true \
  -Dtests.locale=ar-TN \
  -Dtests.timezone=Europe/Sarajevo
```

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+matrix-java-periodic/ES_BUILD_JAVA=java10,ES_RUNTIME_JAVA=java8fips,nodes=virtual&&linux/217/console

--- a/server/src/test/java/org/elasticsearch/action/admin/indices/settings/put/UpdateSettingsRequestStreamableTests.java
+++ b/server/src/test/java/org/elasticsearch/action/admin/indices/settings/put/UpdateSettingsRequestStreamableTests.java
@@ -41,7 +41,7 @@ public class UpdateSettingsRequestStreamableTests extends AbstractStreamableTest
         List<Runnable> mutators = new ArrayList<>();
         mutators.add(() -> mutation
                 .masterNodeTimeout(randomValueOtherThan(request.masterNodeTimeout().getStringRep(), ESTestCase::randomTimeValue)));
-        mutators.add(() -> mutation.timeout(randomValueOtherThan(request.masterNodeTimeout().getStringRep(), ESTestCase::randomTimeValue)));
+        mutators.add(() -> mutation.timeout(randomValueOtherThan(request.timeout().getStringRep(), ESTestCase::randomTimeValue)));
         mutators.add(() -> mutation.settings(mutateSettings(request.settings())));
         mutators.add(() -> mutation.indices(mutateIndices(request.indices())));
         mutators.add(() -> mutation.indicesOptions(randomValueOtherThan(request.indicesOptions(),

